modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-oci-format_{context}"]
+= Mirroring images in OCI format
+
+You can use the oc-mirror plug-in to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can then copy local OCI images to your target mirror registry.
+
+:FeatureName: Using the oc-mirror plug-in to mirror Operator images in OCI format
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+
+.Procedure
+
+. Create the image set configuration file to retrieve the catalogs and images that you require.
++
+.Example image set configuration file for copying to disk
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+ operators:
+ - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+   packages:
+   - name: aws-load-balancer-operator
+----
++
+[NOTE]
+====
+When using the OCI feature, the `storageConfig` setting is ignored in favor of the location passed in to the `oc mirror` command.
+====
++
+// TODO: I removed "helm" from these examples, that okay?
+
+. Run the `oc mirror` command to mirror the images from the specified image set configuration to disk:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \ <1>
+  --use-oci-feature \                         <2>
+  --oci-feature-action=copy \                 <3>
+  oci://my-oci-catalog                        <4>
+----
+<1> Pass in the image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.
+<2> Use the `--use-oci-feature` flag to enable the OCI feature.
+<3> Set the `--oci-feature-action` flag to `copy` to copy the catalog to disk.
+<4> Specify a directory on disk where you want to output the catalog. This procedure assumes that it is named `my-oci-catalog`. The path must start with `oci://`. This directory is created in the current working directory where the `oc mirror` command is run.
++
+// TODO: Should this always be just a directory, or would a full path be okay?
++
+[NOTE]
+====
+You can optionally use the `--oci-registries-config` flag to specify the path to a TOML-formatted `registries.conf` file. You can use this to mirror from a different registry, such as a pre-production location for testing, without having to change the image set configuration file.
+
+.Example registries.conf file
+[source,toml]
+----
+[[registry]]
+ location = "production.registry.io:5000"
```

Comment:
@sherine-k @lmzuccarelli What should this value be based on my previous image set configuration file? `registry.redhat.io`? What about a port?

Reply:
@bergerhoffer - the location will point to a registry. Default will be 443 (https) however if a custom registry is used i.e local docker registry server the default port is 5000, but can be configured to any value really. This is up to the sys-admin to etc.

Reply:
Based on your example (imagesetconfig right above this), location here should be registry.redhat.io, or further registry.redhat.io/redhat